lib/processLayerStyles.js

```diff
@@ -37,10 +37,16 @@ const processLayerStyles = async (layerStyles, filepath, outputFolder, modifiers
         // Border is aplied after the cornerRadius because we need to know the radius for the border
         if (currentModifier.modifier != null) {
           const radius = cornerRadius(currentModifier.modifier)
-          addOptionalProperty("overlay", data.style, layerBorder(layerStyle, radius))
-
-          // Clip the shape for the corner radius
-          addOptionalProperty("clipShape", data.style, cornerRadius(currentModifier.modifier) )
+          // If there is a radius we overlay it and clip it
+          if (radius) {
```

Comment:
this check would be great for swift, but for javascript there is all kinds of falsey / truthy shizzle to deal with. What does cornerRadius return if it doesn't have one? Undefined or null presumably - can the check specifically check for that and for radius == 0?

Reply:
Sure, I can check for `null` since radius is an object